devlib/target.py

```diff
@@ -1530,28 +1530,6 @@ def clear_logcat(self):
     def get_logcat_monitor(self, regexps=None):
         return LogcatMonitor(self, regexps)
 
-    def adb_kill_server(self, timeout=30):
```

Comment:
Some of these should still be exposed via `AndroidTarget` (by calling through to the connection method)?

Reply:
Updated. `wait_for_device` and `reboot_bootloader` are now exposed in AndroidTarget
as generic methods.